prover/crates/bin/prover_fri/tests/basic_test.rs

```diff
@@ -65,7 +66,13 @@ async fn prover_and_assert_base_layer(
     .context("generate_cpu_base_layers_setup_data()")?;
     let setup_data = Arc::new(circuit_setup_data.into());
     let setup_key = ProverServiceDataKey::new(circuit_id, aggregation_round.into());
-    let prover_job = ProverJob::new(block_number, expected_proof_id, circuit_wrapper, setup_key);
+    let prover_job = ProverJob::new(
+        L2ChainId::zero(),
+        block_number,
```

Comment:
same point as above - why have the two fields alongside each other when we introduced `ChainAwareL1BatchNumber`?